src/policies/sessionReport.js

```diff
@@ -55,12 +55,8 @@ export default class SessionReport {
   }
 
   canUpdate() {
-    if (!this.canWriteInRegion()) { return false; }
-
     if (this.isAdmin()) { return true; }
-    // if (this.isCollaborator()) { return true; }
     if (this.isAuthor()) { return true; }
```

Comment:
POCs can edit all sessions on an event.

Reply:
Please think about it this way:

- An owner can add collabs & POC to an event
- At that point, all 3 can create sessions, edit event, and edit all sessions.

Reply:
Ok this has been updated.

Reply:
A POC should be able to edit an event:

![Screenshot 2023-07-13 at 4 48 30 PM](https://github.com/HHS/Head-Start-TTADP/assets/3288586/ec911322-c812-4332-a602-5abb3a97d366)

Reply:
Other issues I see trying to edit as a POC:
- /training-report-users/ now throws a 403 so when editing an event as POC you'll see ```Error fetching collaborators and points of contact```
- /session-reports/eventId/1 also throws a 403 so when editing an event as POC you'll see ```Unable to load sessions```

What we want is:
- An owner can add collabs & POC to an event
- At that point, all 3 (owner, POC, and collabs) can create sessions, edit event, and edit all sessions.